docker/pim-declare-volumes.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+VOLUME ["/home/docker/pim", "/var/log/apache2", "/etc/apache2"]
```

Comment:
I think you should give an example in the README about how to take advantage of the VOLUME you set up